pkg/util/util.go

```diff
@@ -0,0 +1,80 @@
+package util
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/golang/gddo/httputil/header"
+	"io"
+	"net/http"
+	"strings"
+)
+
+const MaxBytesBody = 1048576
+
+type MalformedRequest struct {
+	Status int
+	Msg    string
+}
+
+func (mr *MalformedRequest) Error() string {
+	return mr.Msg
+}
+
+func DecodeJSONBody(w http.ResponseWriter, r *http.Request, dst interface{}) error {
```

Comment:
We should probably write tests for this at some point

Reply:
Created issue for this #18